src/styles/main.scss

```diff
@@ -1,3 +1,80 @@
-body {
-  margin: 0;
+$padding: 10px;
+$gap: 1px;
+$columns: 7;
+$cellSize: 100px;
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: flex-start;
+  padding: $padding;
+  max-width: ($cellSize + $gap) * ($columns + 1);
```

Comment:
You need to adjust (and simplify) your calculations:

- change this property to width
- you need 7 cellSize and 6 gaps